duties/constants/logging.py

```diff
@@ -10,3 +10,4 @@
 SYSTEM_EXIT_MESSAGE = "Detected user intervention (SIGINT). Shutting down."
 NEXT_INTERVAL_MESSAGE = "Printing next interval..."
 NO_UPCOMING_DUTIES_MESSAGE = "No upcoming duties detected!"
+TOO_MANY_PROVIDED_VALIDATORS_MESSAGE = "Provided number of validator indices is higher than 300. This surpasses the current maximum for fetching attestation and sync committee duties. Checking for those duties will be skipped!"  # pylint: disable=line-too-long
```

Comment:
Use backslash to define string in multiple lines. This avoids linter exception definition.

```
TOO_MANY_PROVIDED_VALIDATORS_MESSAGE = "Provided number of validator indices indices is higher than 300." \
"This surpasses the current maximum for fetching attestation and sync committee duties." \
"Checking for those duties will be skipped!"
```

Reply:
Changed the pythonic way 😉